### PR TITLE
fix(workflow): Use incidents+perf flag for alert wizard

### PR DESCRIPTION
**Pull request by scttcper**
disable everything except issues for users without the `incidents` flag

image

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
Hmm, wouldn't the upsell be better if we left these on though, and they just can't create them?

**Comment by EvanPurkhiser** [comment]
oh nvm I see

**Comment by EvanPurkhiser** [comment]
It is like that

**Comment by scttcper** [comment]
@EvanPurkhiser yeah they can use the other radio buttons but they get the "learn more" button from the upsell added yesterday. Just switching flags. Everyone seems to already have performance-view?

**Comment by EvanPurkhiser** [comment]
Old mm2 plans don't have performance.

**Comment by scttcper** [comment]
@EvanPurkhiser should i use both flags? they need metric alerts and performance view

File: static/app/views/alerts/wizard/index.tsx [hunk 1]
```diff
@@ -54,7 +53,7 @@ class AlertWizard extends React.Component<Props, State> {
     const {organization, project, location} = this.props;
     const {alertOption} = this.state;
     const metricRuleTemplate = AlertWizardRuleTemplates[alertOption];
-    const isTransactionDataset = metricRuleTemplate?.dataset === Dataset.TRANSACTIONS;
+    const isMetricAlert = !!metricRuleTemplate?.dataset;
```
Comment (davidenwang) [inline static/app/views/alerts/wizard/index.tsx:56]: nit: can just use `!!metricRuleTemplate`
```suggestion
    const isMetricAlert = !!metricRuleTemplate;
```

**Review by davidenwang — APPROVED** [review]
looks good code wise, but i am also a little unsure about the feature flags. If incidents implies performance and metric alerts then we should be good, but it seems like we might need to do separate checks for metric alerts and then transaction metric alerts.

**Pull request merged**